sdk/src/events/fetchLogs.ts

```diff
@@ -36,7 +36,8 @@ export async function fetchLogs(
 	finality: Finality,
 	beforeTx?: TransactionSignature,
 	untilTx?: TransactionSignature,
-	limit?: number
+	limit?: number,
+	batchSize = 25
```

Comment:
would be good for the EventSubscriber to have an option for this when polling is configured: https://github.com/drift-labs/protocol-v2/blob/master/sdk/src/events/types.ts#L137